src/community/ogcapi/dggs/dggs-core/src/main/java/org/geotools/dggs/h3/H3DGGSInstance.java

```diff
@@ -367,14 +367,4 @@ public Filter getChildFilter(FilterFactory2 ff, String zoneId, int resolution, b
         // return ff.and(prefixFilter, matchFilter);
         return matchFilter;
     }
-
```

Comment:
If this is removed, then the above code should be also removed rather than commented out.

Reply:
I backed out this change.